Misc/NEWS.d/next/Library/2023-04-12-03-03-27.gh-issue-103464.Oa_8IW.rst

```diff
@@ -0,0 +1 @@
+Add arguments checking for :mod:`pdb` commands
```

Comment:
Just a suggestion to make this improvement "pop" a little more for anyone skimming the changelog:

```suggestion
Provide helpful usage messages when parsing incorrect :mod:`pdb` commands.
```